src/app/vendors/Vendors.tsx

```diff
@@ -182,6 +184,29 @@ function Vendors({
     vendorGroups = filterVendorGroupsToSearch(vendorGroups, searchQuery, filterItems);
```

Comment:
@bhollis can you confirm why there's two `vendorGroups &&` (L180, L183)?
is there an undocumented possibility of undefinedness?